web/src/components/molecules/Schema/FieldModal/FieldCreationModal/index.tsx

```diff
@@ -139,7 +146,7 @@ const FieldCreationModal: React.FC<Props> = ({
           };
         } else if (selectedType === "Integer") {
           values.typeProperty = {
-            integer: { defaultValue: +values.defaultValue, min: +values.min, max: +values.max },
+            integer: { defaultValue: values.defaultValue, min: +values.min, max: +values.max },
```

Comment:
Is there a reason you are adding `+` to min/max values?

Reply:
@KaWaite , it's an operator to convert the values to numbers.